src/main/java/org/folio/oaipmh/helpers/AbstractGetRecordsHelper.java

```diff
@@ -106,7 +106,7 @@ protected void requestAndProcessSrsRecords(Request request, Context ctx, Promise
             }
           });
         } else {
-          logger.error(request.getVerb().value() + " response from SRS status code: {0}: {1}", response.statusMessage(), response.statusCode());
+          logger.error(request.getVerb().value() + " response from SRS status code: {}: {}", response.statusMessage(), response.statusCode());
```

Comment:
Where this is an error it's unlikely to make a big difference, but generally its better to use lazy logging...

```
logger.error("{} response from SRS status code: {}: {}",
    () -> request.getVerb().value(),
    () -> response.statusMessage(),
    () -> response.statusCode());
```

Reply:
Okay, thank you for the advice!